### PR TITLE
Implementation game

### PR DESCRIPTION
**Pull request by TarikElha**

### CODE REVIEW

File: src/View/Card/show.html.twig [hunk 1]
```diff
@@ -1,13 +1,16 @@
 {% extends 'layout.html.twig' %} {% block content %}
 <h1 class="unlock">UNLOCK !</h1>
 <h2 class="unlock-title2">Sherlock Adventure</h2>
+
 <div class="card card-detail">
     <div class="card-detail-title-container">
         <img src="/assets/images/card-icon.png" alt="numéro de carte" class="card-number">
         <p>{{card.number}}</p>
         <h2 class="card-name ">{{ card.name }}</h2>
```
Comment (MarjorieThomassin) [inline src/View/Card/show.html.twig:9]: only 1 h2 on section :)

**Review by MarjorieThomassin — CHANGES_REQUESTED** [review]

File: public/assets/css/style.css [hunk 2]
```diff
@@ -301,6 +301,11 @@ button {
 }
 
 
+.card-image{
+    margin : auto;
+    width:100%;
+    height:100%;
+
```
Comment (Cyl07) [inline public/assets/css/style.css:308]: Ca vas tout casser sans accolade

File: src/View/Card/show.html.twig [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
     </div>
     <img src="/assets/images/key.png" alt="Image de la carte" class="card-img">
+
     <p class="card-description">{{ card.description }}</p>
+    <br>
```
Comment (Cyl07) [inline src/View/Card/show.html.twig:14]: br inside of paragraphe only use CSS to position your card-description

File: populate.sql [hunk 4]
```diff
@@ -6,7 +6,18 @@ INSERT INTO `game` (`name`, `description`, `instruction`) VALUES
...
+("Cable", "Voici un cable téléphonique.", ""),
+("Clé", "C'est une clé, qui permet sûrement d'ouvrir une serrure...", ""),
+("Armoire", "Une armoir fermée à clé.", ""),
+("Watson", "Il vous connait bien, il peut peut-être vous aider.", ""),
+("Porte", "La porte d'entrée est fermée à clé.", ""),
+("Carnet téléphonique", "Ce carnet contient le numéro de téléphone de Watson.", ""),
+("Téléphone", "Appel à un ami.", ""),
+("Livre", "Ce livre contient une clé.", "");
+
+/*INSERT INTO `card` (`name`, `description`) VALUES
```
Comment (Cyl07) [inline populate.sql:20]: delete comment

**Review by Cyl07 — CHANGES_REQUESTED** [review]

**Pull request merged**